src/pages/AppConfigure.vue

```diff
@@ -107,6 +108,7 @@ function handleSmartContractErrors(type: string, error: unknown) {
 
 function handleCancel() {
   appStore.fetchAppConfig()
+  createTransactionSigner()
```

Comment:
Why do we need to fetch the app config and re-initialised the transaction signer again here? It should already be initialised to the current app.

Reply:
We need to fetch app config to reset the store with existing config. Currently all the changes made in the configure page changes the state of the app.store, so on cancel we need to reset it in order to be in sync with gateway values.

Reply:
Okay, let's keep it until we move to using draft states for forms.